src/problem/Problem.cpp

```diff
@@ -21,15 +21,17 @@ namespace polyfem
 	{
 	}
 
-	void Problem::setup_bc(const Mesh &mesh, const std::vector<ElementBases> &bases, std::vector<LocalBoundary> &local_boundary, std::vector<int> &boundary_nodes, std::vector<LocalBoundary> &local_neumann_boundary)
+	void Problem::setup_bc(const Mesh &mesh, const std::vector<ElementBases> &bases, const std::vector<ElementBases> &pressure_bases, std::vector<LocalBoundary> &local_boundary, std::vector<int> &boundary_nodes, std::vector<LocalBoundary> &local_neumann_boundary, std::vector<int> &pressure_boundary_nodes)
```

Comment:
this affects only splitting right?

Reply:
Yes, this `pressure_boundary_nodes` is defined only to set up the pressure dirichlet bc for splitting. It's constructed in this function, same as what you do to construct the `boundary_nodes`.